data/rhea_db/meta.yaml

```diff
@@ -0,0 +1,45 @@
+---
+name: rhea_db
+description: |-
+    Rhea is a freely available, manually annotated database of chemical reactions,
+    created by the Swiss Institute of Bioinformatics (SIB) and used as source
+    of chemical reactions for the UniProt Knowledgebase (UniProtKB) annotation.
+    This dataset contains the reactions from Rhea database and their components
+    in smiles format and ChEBI IDs.
+    Tasks can be formulated as translation between smiles and english and vice
+    versa, or as a completion of stechiometric coefficients or masked reactants.
+targets:
+    - id:
+      description:
+      units:
+      type:
+identifiers:
+    - id: rhea_id
+      type: int
+      description: Rhea reaction ID
```

Comment:
add info that this is not sampled for the prompt @MicPie

Reply:
done